syft/pkg/cataloger/common/cpe/java_groupid_map.go

```diff
@@ -1,250 +1,320 @@
+	"elasticsearch-h3":                  "org.elasticsearch",
+	"elasticsearch-hadoop":              "org.elasticsearch",
+	"elasticsearch-hadoop-cascading":    "org.elasticsearch",
+	"elasticsearch-hadoop-hive":         "org.elasticsearch",
+	"elasticsearch-hadoop-mr":           "org.elasticsearch",
+	"elasticsearch-hadoop-pig":          "org.elasticsearch",
+	"elasticsearch-lang-groovy":         "org.elasticsearch",
+	"elasticsearch-lang-javascript":     "org.elasticsearch",
+	"elasticsearch-lang-mvel":           "org.elasticsearch",
+	"elasticsearch-lang-python":         "org.elasticsearch",
+	"elasticsearch-logging":             "org.elasticsearch",
+	"elasticsearch-lz4":                 "org.elasticsearch",
+	"elasticsearch-mapper-attachments":  "org.elasticsearch",
+	"elasticsearch-nio":                 "org.elasticsearch",
+	"elasticsearch-parent":              "org.elasticsearch",
+	"elasticsearch-plugin-analysis-api": "org.elasticsearch",
+	"elasticsearch-plugin-api":          "org.elasticsearch",
+	"elasticsearch-plugin-classloader":  "org.elasticsearch",
+	"elasticsearch-plugin-scanner":      "org.elasticsearch",
+	"elasticsearch-preallocate":         "org.elasticsearch",
+	"elasticsearch-repository-hdfs":     "org.elasticsearch",
+	"elasticsearch-river-couchdb":       "org.elasticsearch",
+	"elasticsearch-river-rabbitmq":      "org.elasticsearch",
+	"elasticsearch-river-twitter":       "org.elasticsearch",
+	"elasticsearch-river-wikipedia":     "org.elasticsearch",
+	"elasticsearch-secure-sm":           "org.elasticsearch",
+	"elasticsearch-spark-1.2":           "org.elasticsearch",
+	"elasticsearch-spark-1.2_2.10":      "org.elasticsearch",
+	"elasticsearch-spark-1.2_2.11":      "org.elasticsearch",
+	"elasticsearch-spark-13_2.10":       "org.elasticsearch",
+	"elasticsearch-spark-13_2.11":       "org.elasticsearch",
+	"elasticsearch-spark-20_2.10":       "org.elasticsearch",
+	"elasticsearch-spark-20_2.11":       "org.elasticsearch",
+	"elasticsearch-spark-20_2.12":       "org.elasticsearch",
+	"elasticsearch-spark-30_2.12":       "org.elasticsearch",
+	"elasticsearch-spark-30_2.13":       "org.elasticsearch",
+	"elasticsearch-spark_2.10":          "org.elasticsearch",
+	"elasticsearch-spark_2.11":          "org.elasticsearch",
+	"elasticsearch-ssl-config":          "org.elasticsearch",
+	"elasticsearch-storm":               "org.elasticsearch",
+	"elasticsearch-tdigest":             "org.elasticsearch",
+	"elasticsearch-transport-memcached": "org.elasticsearch",
+	"elasticsearch-transport-thrift":    "org.elasticsearch",
+	"elasticsearch-transport-wares":     "org.elasticsearch",
+	"elasticsearch-x-content":           "org.elasticsearch",
+	"elasticsearch-yarn":                "org.elasticsearch",
+	"elasticsearch-rest-client":         "org.elasticsearch.client", - -"elasticsearch-rest-client-sniffer": "org.elasticsearch.client", - -"elasticsearch-rest-high-level-client": "org.elasticsearch.client",
```

Comment:
It looks like some diff output ended up in here or something @westonsteimel

Reply:
Thanks, that piece should be fixed now